src/ckeditor/smartpicker/InsertItem.js

```diff
@@ -0,0 +1,81 @@
+/**
```

Comment:
Please don't forget to delete this file ;)

Reply:
Must've did something wrong while squashing. Thank you for the heads up